amd_openvx/openvx/include/VX/vx_compatibility.h

```diff
@@ -160,9 +160,6 @@
 #define VX_THRESHOLD_FALSE_VALUE (VX_ATTRIBUTE_BASE(VX_ID_KHRONOS, VX_TYPE_THRESHOLD) + 0x5)
 #define VX_THRESHOLD_DATA_TYPE (VX_ATTRIBUTE_BASE(VX_ID_KHRONOS, VX_TYPE_THRESHOLD) + 0x6)
 
-#define VX_BIDIRECTIONAL (VX_ENUM_BASE(VX_ID_KHRONOS, VX_ENUM_DIRECTION) + 0x2)
-
-
```

Comment:
@hansely - we should not add anything into the VX/ folder. We can add these in [vx_ext_amd.h](https://github.com/ROCm/MIVisionX/blob/develop/amd_openvx/openvx/include/vx_ext_amd.h)